main.py

```diff
@@ -54,10 +70,72 @@ def configure_confluent_kafka_consumer(event, args):
         boto3.set_stream_logger()
         logger.debug(f"Using boto3 {boto3.__version__}")
```

Comment:
Lines 71-79 should be outdented. `logger.debug()` calls already only occur if debug level is set.  The other lines all need to be run regardless of log level.

Also, whilst you're making changes, could you put the imports at the top of this script in alphabetical order please?